src/Symfony/Component/Console/Color.php

```diff
@@ -116,6 +116,10 @@ private function parseColor(string $color, bool $background = false): string
             return '';
         }
 
+        if (0 === strpos($color, 'rgb(')) {
```

Comment:
You should use `str_starts_with`.

Reply:
I would have loved to! But minimum requirement for Symfony 5.4 is PHP 7.2, and these methods are only available since PHP 8.

Reply:
Console component uses PHP 8 polyfill.
This means you can use PHP 8 functions even if you are running on a lower version.
https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Console/composer.json#L23

Reply:
Nice, good to know! I made the change. Thanks!